generate_pa11y_ci_urls_from_git_diff.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Take in a base ref. Gets the files that have changed from the base to the HEAD.
+# Export the list as a CommonJS module for the pa11y-ci configuration to use.
+# Typical usage: generate_pa11y_ci_urls_from_git_diff.sh gh-pages > .pa11yci.js
+
+function generate_pa11y_ci_urls_from_git_diff {
+  base_ref=$1
+  if [ -z "$base_ref" ]; then
+    echo "No Git base ref provided. Usage: $0 <base_ref>"
+    exit 1
+  fi
+
+  changed_files=$(git diff --name-only $base_ref)
+
+  echo -n "module.exports=[";
+  for file in $changed_files; do
+    if [[ $file == _posts/* ]]; then
+      echo -n "'/$(url_of_post $file)',";
+    elif [[ $file == *.html && ! $file == _includes/* && ! $file == _layouts/* ]]; then
```

Comment:
These folders contain HTML templates, but they don't represent webpages. They represent excerpts (in the case of `_includes`) or layouts (in the case of `_layouts`). Since they don't correspond to webpages on the blog, I can't automatically determine which webpages will change as a consequence of changes in these files, short of determining the semantics of where these excerpts/layouts get used (which I don't want to do because I'm trying to do something cheap & simple).